src/sentry/utils/sdk_crashes/sdk_crash_detector.py

```diff
@@ -32,7 +32,7 @@ def is_sdk_supported(
             return False
 
         try:
-            minimum_sdk_version = Version(self.config.min_sdk_version)
+            minimum_sdk_version = Version(self.config.sdk_names[sdk_name])
```

Comment:
Any chance of this crashing because the `sdk_name` is not in the `sdk_names` list?

Reply:
No because I check that above, but I changed the code to make it more explicit it doesn't happen.